.redhat-ci.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# F25 currently has 2.2.1, so install from pypi
+pip install ansible==2.2.2.0
+
+# do a simple ping to make sure the nodes are available
+ansible -vvv -i .redhat-ci.inventory nodes -a 'rpm-ostree status'
+
+upload_journals() {
+  mkdir journals
+  for node in master node1 node2; do
+    ssh ocp-$node 'journalctl --no-pager || true' > journals/ocp-$node.log
```

Comment:
Why not use ansible and the inventory file here?
If we change the inventory, this code gets out of sync.